jsdoc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "source": {
+    "exclude": [
+      "__snapshots__",
+      "build",
+      "coverage",
+      "logo",
+      "node_modules",
+      "private",
+      "public"
+    ]
+  },
+  "plugins": ["plugins/markdown"],
+  "opts": {
+    "destination": "api-docs/",
+    "readme": "README.md",
+    "recurse": true,
+    "tutorials": "doc/"
```

Comment:
This feature is really nice because it means we can embed our markdown tutorials in with the API docs.
Unfortunately the logo tutorial has one line that renders a bit odd. Its not the end of the world, and we can probably sort that with a bit of custom CSS, but I'll wait and see what we do about theming first.